bpf-map-pressure-exporter/TAG

```diff
@@ -0,0 +1 @@
+0.1.0
```

Comment:
I think it's good as `1.0.0`

Reply:
https://github.com/cybozu/neco-containers/pull/1155/commits/00bcfdd38f8e727229474029a4270aa03d340dc2